src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    - For a client, it's mandatory if the ssl_ca_file parameter is set for a
+    server; otherwise, optional.
+]])
+
+M['<uri>.params.ssl_ciphers'] = format_text([[
+    (Optional) A colon-separated (:) list of SSL cipher suites the connection
+    can use. Note that the list is not validated: if a cipher suite is unknown,
+    Tarantool ignores it, doesn't establish the connection, and writes to the
+    log that no shared cipher was found.
+]])
+
+M['<uri>.params.ssl_key_file'] = format_text([[
+    A path to a private SSL key file:
+
+    - For a server, it's mandatory.
+    - For a client, it's mandatory if the `ssl_ca_file` parameter is set for a
+    server; otherwise, optional.
+
+    If the private key is encrypted, provide a password for it in the
+    `ssl_password` or `ssl_password_file` parameter
+]])
+
+M['<uri>.params.ssl_password'] = format_text([[
+    (Optional) A password for an encrypted private SSL key provided using
+    `ssl_key_file`. Alternatively, the password can be provided in
+    `ssl_password_file`.
+
+    Tarantool applies the `ssl_password` and `ssl_password_file` parameters in
+    the following order:
+
+    - If `ssl_password` is provided, Tarantool tries to decrypt the private key
+    with it.
+    - If `ssl_password` is incorrect or isn't provided, Tarantool tries all
+    passwords from `ssl_password_file` one by one in the order they are written.
+    - If `ssl_password` and all passwords from `ssl_password_file` are
+    incorrect, or none of them is provided, Tarantool treats the private key as
+    unencrypted.
+]])
+
+M['<uri>.params.ssl_password_file'] = format_text([[
+    (Optional) A text file with one or more passwords for encrypted private
+    SSL keys provided using `ssl_key_file` (each on a separate line).
+    Alternatively, the password can be provided in `ssl_password`.
+]])
+
+M['<uri>.params.transport'] = format_text([[
+    Allows you to enable traffic encryption for client-server communications
+    over binary connections. In a Tarantool cluster, one instance might act
+    as the server that accepts connections from other instances and the client
+    that connects to other instances.
+
+    `<uri>.params.transport` accepts one of the following values:
+
+    - `plain` (default): turn off traffic encryption,
+    - `ssl`: encrypt traffic by using the TLS 1.2 protocol (EE only).
+]])
+
+-- }}} <uri>.params configuration
+
+-- {{{ app configuration
+
+M['app'] = format_text([[
+    Using Tarantool as an application server, you can run your own Lua
+    applications. In the `app` section, you can load the application and
+    provide an application configuration in the `app.cfg` section.
+]])
+
+M['app.cfg'] = format_text([[
+    A configuration of the application loaded using `app.file` or `app.module`.
+]])
+
+M['app.cfg.*'] = format_text([[
+    Specify `file` (a path to a Lua file to load an application from) or
+    `module` (a Lua module to load an application from).
+]])
```

Comment:
This mapping has arbitrary user-defined values. `file` and `module` fields are in `app`.

Reply:
Maybe like this?
```lua
I['app.cfg.*'] = format_text([[
    Mapping for arbitrary user-defined configuration values, accessible in
    the application via `require('config'):get('app.cfg.*')`.
]])
```
Tarantool documentation does not contain a description of the `app.cfg.*` field, only an example of usage.

Reply:
Generally OK, a few remarks:

* I would eliminate `require` (usually it is on some seperate line on top of a module).
* I would use `app.cfg.foo` or kinda, because the asterisk is not supported in `:get()`.

Result:

> <...> via `config:get('app.cfg.foo')`.

Or we can show just `app.cfg` as the example:

> <...> via `config:get('app.cfg')`.

Reply:
Replaced with `config:get('app.cfg')`